release/release.go

```diff
@@ -927,7 +927,7 @@ func uploadReleaseJSON(v version.Version) {
 			continue
 		}
 
-		if _, ok := platform.GetByVariant(task.Variant); !ok {
+		if p, ok := platform.GetByVariant(task.Variant); p.SkipForJSONFeed || !ok {
```

Comment:
Now we'll have one less sign task, but `platform.Count()` will still be returning the number all the platforms right? So I think `len(signTasks) != pfCount` will return true.

Reply:
Oh, I think I misunderstood the question. I think you're right that this breaks the count check.

Reply:
Fixed.